app/models/miq_server/environment_management.rb

```diff
@@ -29,7 +29,7 @@ def minimal_env_options
         options.shift # remove the "minimal" from the front of the array
 
         # Special case where Netbeans is handling the UI worker for debugging
-        options.collect { |o| o.downcase == "netbeans" ? %w(schedule reporting noui) : o }.flatten
+        options.collect { |o| o.downcase == "netbeans" ? %w(schedule reporting no_ui) : o }.flatten
```

Comment:
@chessbyte I still use this with RubyMine, but it could be changed to `ui_only` maybe?  I think that's what it means, although I'm pretty sure it can be combined with others, so maybe something else like just `ui` might be more appropriate.  This might remove the need for `no_ui` as its absence would indicate to not run it.